build.gradle.kts

```diff
@@ -35,3 +35,13 @@ nexusPublishing {
 // tasks.release.finalizedBy tasks.closeAndReleaseRepository
 
 description = "OpenTelemetry instrumentations for Java"
+
+subprojects {
+  if (path != ":instrumentation:liberty:compile-stub" && !path.startsWith(":instrumentation:internal:")) {
```

Comment:
applying it to `:instrumentation:internal:` modules made me put the comment on Advice classes which felt weird, so figured I should at least do that in a separate PR to get feedback separately